app/assets/stylesheets/application.scss

```diff
@@ -1,11 +1,272 @@
+}
+
+
+.top-bar-section li:not(.has-form) a:not(.button) {
+    background: #333333;
+}
+
+.top-bar-section li {
+  text-align: center;
+}
+
+.notice {
+  background-color: $fr-green;
+  text-align: center;
+  color: $sc-white;
+}
+
+.alert {
+  background-color: $focus-red;
+  text-align: center;
+  color: $sc-white;
+}
+
+.mbta {
+  color: $sc-white;
+  padding: 0px;
+  margin: 0px;
+  line-height: auto;
+  font-size: 14px;
+  background-color: $focus-red;
+  padding-left: 40px;
+  margin-left: 90px;
+  margin-right: 90px;
+}
+.mbta-header {
+  color: $focus-red;
+  padding-left: 120px;
+  font-weight: bolder;
+}
+
+.logo {
+  padding-left: 30px;
+}
+
+a {
+  color: $white;
+}
+
+a:hover, a:focus {
+  color: $dark-green;
+}
+
+.flash-alert {
+  margin-top: 10px;
+}
+
+.trip-table {
+  margin-left: 200px;
+  margin-right: 200px;
+  background-color: $fr-green;
+}
+
+.trip-table a {
+  color: $fr-black;
+}
+// .review-form-list {
+//   margin: 150px;
+//   padding: 150px;
+// }
+
+.forgot-pass {
+  color: $light-green;
+}
+
+.forgot-pass a {
+  color: $light-green;
+}
+
+.review-title {
+  margin-bottom: 0rem;
+  padding: none;
+}
+
+.park-index-list-item {
+  background-color: $fr-green;
+  list-style-type: none;
+  margin-bottom: 5px;
+  padding: 10px 15px;
+}
+
+.index-details {
+  color: $steel;
+  float: left;
+  font-size: 100%;
+  line-height: 2;
+}
+
+.index-rating p {
+  color: $steel;
+  float: center;
+  font-size: 300%;
+  line-height: 0.6;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.park-info p {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  border-right: 2px solid;
+  border-color: $white;
+}
+
+.review-info p {
+  border-color: $white;
+  border-right: 2px solid;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.park-icons img {
+  margin-right: 10px;
+  opacity: 0.6;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.review-body p {
+  position: relative;
+  top: 100%;
+  transform: translateY(-50%);
+  background-color: $fr-green;
+}
+
+.park-show-location {
+  margin-top: 0rem;
+}
+
+.park-show-rating {
+  margin-bottom: 1rem;
+}
+
+.show-icons {
+  margin-bottom: 10px;
+}
+
+.top-bar {
+  height: 3rem;
+  background: #333333;
+}
+
+.new {
+  color: $white;
+}
+
+.no-bullet {
+  margin-bottom: 0;
+  margin-left: 0;
+}
+
+.button.small {
+  font-size: 1.2rem;
+  padding: 0.4rem;
+}
+
+.button.tiny {
+  font-size: 0.8rem;
+  padding: 0.4rem 0.6rem;
+}
+
+.button.tiny.vote {
+  margin: 0;
+}
+
+
+.has-form {
+  position: absolute;
+  top: 0;
+  left: 8rem; // adjust distance from left here
+  min-width: 14rem; // adjust width here
+
+  @media only screen and (max-width: 40em) {
+    min-width: 10rem;
+    top: 0.5rem;
+
+    .button {
+      height: 1.85rem;
+    }
+  }
+}
+
+.leaf {
+  background: url('http://static.pexels.com/wp-content/uploads/2014/07/beach-coast-holiday-2190-825x550.jpg') no-repeat center center;
```

Comment:
Prefer double-quoted strings<br>URL `http://static.pexels.com/wp-content/uploads/2014/07/beach-coast-holiday-2190-825x550.jpg` should not contain protocol or domain